pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,253 @@ foo {
 
 </div>
 
+### Trailing commas
+
+A trailing comma is a comma sign after the last item of the element series:
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Person(
+    val name: String,
+    val surname: String,
+    val age: Int, // trailing comma
+)
+```
+
+</div>
+
+Using trailing commas has several benefits:
+
+* It makes version-control diffs cleaner: all focus is on changed value.
+* It's easy to add or reorder elements: no need to add or delete the comma sign if you manipulate elements.
+* It simplifies code generation (for example, for object initializers). The last element can also have the comma sign.
+
+Using trailing commas are optional: your code will work without them. But Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
+
+To enable trailing commas for the IntelliJ formatter, go to __Settings | Editor | Code Style | Kotlin__, 
+open the __Other__ tab and enable the __Use trailing comma__ option.
+
+Kotlin supports trailing commas in the following cases:
+* [Lists, enumerations, value arguments](#lists-enumerations-value-arguments)
+* [Class parameters](#class-parameters)
+* [Function value parameters](#function-value-parameters)
+* [Parameters with optional type (including setters)](#parameters-with-optional-type-including-setters)
+* [Indexing suffix](#indexing-suffix)
+* [Lambda parameters](#lambda-parameters)
+* [`when` entry](#when-entry)
+* [Collection literals (in annotations)](#collection-literals-in-annotations)
+* [Type arguments and type parameters](#type-arguments-and-type-parameters)
+* [Destructuring declarations](#destructuring-declarations)
+
+#### Lists, enumerations, value arguments
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+val colors = listOf(
+    "red",
+    "green",
+    "blue", // trailing comma
+)
+
+enum class Direction {
+    NORTH,
+    SOUTH,
+    WEST,
+    EAST, // trailing comma
+}
+
+fun sum(x: Any, y: Any) {}
+
+fun main() {
+    sum(
```

Comment:
I think `listOf()` is a more natural example for value parameters (and it doesn't require declaration and `main` :)
```
val numbers = listOf(
    "one",
    "two",
    "three", // trailing comma
)
```